src/main/java/net/syscon/elite/service/BookingService.java

```diff
@@ -634,7 +666,16 @@ public MilitaryRecords getMilitaryRecords(final Long bookingId) {
 
     @Transactional
     public void updateLivingUnit(final Long bookingId, final Long livingUnitId) {
-        // TODO DT-235 implement this
+        final var location = locationService.getLocation(livingUnitId);
+        var offenderBooking = offenderBookingRepository.findById(bookingId)
+                .orElseThrow(new EntityNotFoundException(format("Offender booking for booking id %d not found", bookingId)));
```

Comment:
There are some static factory methods available on the `EntityNotFoundException` class...meh.